client/package.json

```diff
@@ -2,7 +2,8 @@
   "repository": "sharetribe/sharetribe",
   "license": "MIT",
   "engines": {
-    "node": "6.9"
```

Comment:
Remember to update the README.md which currently states that Node 6.9 is required.

Reply:
Installation requirements still state Node 6.9